platform/fabric/core/generic/committer/committer.go

```diff
@@ -306,13 +306,17 @@ func (c *committer) listenTo(txid string, timeout time.Duration) error {
 		iterations = 1
 	}
 	for i := 0; i < iterations; i++ {
+		timeout := time.NewTimer(c.pollingTimeout)
```

Comment:
wasn't this the case where .After is okay?

Reply:
same explanation as the other place